### PR TITLE
Addded docs about pg_service QField configuration and QFieldCloud secrets

### PR DESCRIPTION
**Pull request by suricactus**
image

### CODE REVIEW

File: documentation/how-to/pg-service.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+A `pg_service.conf` file allows to use an named alias for a PostgreSQL server connection. Instead of storing hostname, port, database name and more into the QGIS Project file, these can be stored separately. It is even possible to store username and password in a `pg_service.conf` file, to avoid having this stored in clear text in the QGIS Project.
+
+Read more about PostgreSQL services in the [QGIS documentation](https://docs.qgis.org/3.22/en/docs/user_manual/managing_data_source/opening_data.html#postgresql-service-connection-file).
+
+# QField - Direct Connection
+
+If you directly connect from QGIS to your database you can make use of a `pg_service.conf` file by placing it in the QField data folder. You can place your file either on the Internal Device Storage or on the SD Card Storage. You can check the path for the QField data folder in the bottom of the `About QField` screen in the app.
+
+Usually the path on Android devices looks something like this: `/Android/data/ch.opengis.qfield/files/QField`.
```
Comment (m-kuhn) [inline documentation/how-to/pg-service.md:15]: Do we have a canonical place for this doc?

Comment (suricactus) [reply documentation/how-to/pg-service.md:15]: This is a good candidate to be rewritten and serve as common place for this docs: https://docs.qfield.org/how-to/outside-layers/#add-a-layer-outside-of-exported-project-folder However, I think it is out of scope of this PR.

Comment (m-kuhn) [reply documentation/how-to/pg-service.md:15]: Can you open an issue and then we merge?

Comment (suricactus) [reply documentation/how-to/pg-service.md:15]: here https://github.com/opengisch/QField-docs/issues/217

**Review by marioba — APPROVED** [review]

**Pull request merged**